h2/src/docsrc/help/help.csv

```diff
@@ -4296,7 +4296,7 @@ Replaces each substring that matches a regular expression.
 For details, see the Java ""String.replaceAll()"" method.
 If any parameter is null (except optional flagsString parameter), the result is null.
 
-Flags values limited to 'i', 'c', 'n', 'm'. Other symbols causes exception.
+Flags values limited to 'i', 'c', 'n', 'm'. Other symbols causes exception (except for 'g' which is ignored when running in PostgreSQL mode).
```

Comment:
Usually we don't describe such deviations in documentation of functions.

Here is a better place to describe it:
https://github.com/h2database/h2database/blob/e8f99400a3f71312f8edbc86f088becb67210cf5/h2/src/docsrc/html/features.html#L940-L957

Reply:
Thanks, will add something there.